cdap-data-fabric/src/main/java/co/cask/cdap/data2/registry/UsageRegistry.java

```diff
@@ -57,7 +58,42 @@ public UsageDatasetIterable get() {
   }
 
   /**
+   * Registers usage of a stream by multiple ids.
+   *
+   * @param owners the owners of the stream
+   * @param streamId the stream
+   */
+  public void register(List<Id> owners, Id.Stream streamId) {
```

Comment:
No, neither should be null. Would've added `@Nullable` otherwise. Not sure if renaming to `registerAll` adds much since it takes `List<Id>`.